src/main/resources/uischema/smb.json

```diff
@@ -0,0 +1,146 @@
+{
+  "connectorName": "file",
+  "connectionName": "SMB2",
```

Comment:
We will need to add "SMB" connection type for the UI schema of the operations. Please check https://github.com/wso2-extensions/esb-connector-file/blob/master/src/main/resources/uischema/write.json